website/content/partials/known-issues/transit-managed-keys-panics.mdx

```diff
@@ -0,0 +1,16 @@
+### Transit Encryption with Cloud KMS managed keys causes a panic
+
+#### Affected versions
+
+- 1.13.1+ up to 1.13.7 inclusively
+- 1.14.0+ up to 1.14.3 inclusively.
+
+#### Issue
+
+Vault will panic when a Transit encryption API call is backed by a Cloud KMS managed key (Azure, GCP, AWS).
+
+<Note>Encryption and decryption APIs are not affected with PKCS#11 managed keys and non-managed key, key types</Note>
```

Comment:
```suggestion
<Note>

The issue does not affect encryption and decryption with the following key types:

- PKCS#11 managed keys
- non-managed keys

</Note>
```

Style correction: make important information easy to scan and parse

Reply:
Applied in https://github.com/hashicorp/vault/pull/23080/commits/73a206f089239be8eee21737a8db955c6bbe762b, just tweaked non-managed keys to Transit native keys